<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KaibaoTotolClass</class>
 <widget class="QDialog" name="KaibaoTotolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KaibaoTotol</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="lineEdit_filepath"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_selfile">
       <property name="text">
        <string>选择文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_export">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>导出图片</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>场地：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_title"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>年：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_year"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>月：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_month"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget_chart" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_12">
      <property name="leftMargin">
       <number>25</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>25</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_title">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(246, 172, 152);</string>
          </property>
          <property name="text">
           <string>标题</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,1,1,1">
          <item>
           <widget class="QLabel" name="label_year">
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(246, 172, 152);</string>
            </property>
            <property name="text">
             <string>年</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(246, 172, 152);</string>
            </property>
            <property name="text">
             <string>第一场</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(246, 172, 152);</string>
            </property>
            <property name="text">
             <string>第二场</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(246, 172, 152);</string>
            </property>
            <property name="text">
             <string>第三场</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>4</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_totol">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="autoScroll">
         <bool>false</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>4</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>60</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="KaibaoTotol.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_selfile</sender>
   <signal>clicked()</signal>
   <receiver>KaibaoTotolClass</receiver>
   <slot>setfilepath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_export</sender>
   <signal>clicked()</signal>
   <receiver>KaibaoTotolClass</receiver>
   <slot>on_export()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_year</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>KaibaoTotolClass</receiver>
   <slot>on_year_change(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>571</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_month</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>KaibaoTotolClass</receiver>
   <slot>on_month_change(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>571</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setfilepath()</slot>
  <slot>on_export()</slot>
  <slot>on_year_change(QString)</slot>
  <slot>on_month_change(QString)</slot>
 </slots>
</ui>
